Formula/dockly.rb

```diff
@@ -0,0 +1,24 @@
+require "language/node"
+
+class Dockly < Formula
+  desc "Immersive terminal interface for managing docker containers, services and images"
+  homepage "https://github.com/lirantal/dockly#readme"
+  url "https://registry.npmjs.org/dockly/-/dockly-3.21.4.tgz"
+  sha256 "87b2d87ea01eb43fdfd030c775df69249dd2ecae14d4452360d343095cbae943"
+  license "MIT"
+
+  depends_on "node"
+
+  on_linux do
+    depends_on "libx11"
+  end
```

Comment:
Ah, this doesn't help because `npm` probably downloads pre-buit binaries that link with a different `libX11`.

Reply:
Hmm, do you have any idea how we can fix this?

Reply:
We would have to build from source whatever is pre-built and replace the pre-built bits that are bundled by `npm`. How to do that exactly depends on what it is we're trying to replace.

Reply:
How do you recommend debugging this?
Is it possible to run one of the actions in debug/verbose mode?

Reply:
@carlocab I'm really struggling to find a solution for that. do you have any further reading resources/information I could use in order to fix this?